tests/test_proxy.py

```diff
@@ -21,36 +21,24 @@ def setUp(self):
             'url': 'fake-url'
         }
 
-    @mock.patch(
-        "django_docker_engine.proxy.DockerClientWrapper"
-        ".lookup_container_url", return_value="http://example.com"
-    )
-    def test_csrf_exempt_true(self, url_mock):
-        proxy = Proxy(
-            '/tmp/django-docker-engine-test',
-            csrf_exempt=True
-        )
-        urlpatterns = proxy.url_patterns()
-
+    def _check_proxy_csrf(self, csrf_exempt=True):
         with mock.patch("django_docker_engine.proxy.csrf_exempt_decorator") \
-                as csrf_decorator_mock:
-            urlpatterns[0].callback(**self.fake_get_kwargs)
-            self.assertEqual(csrf_decorator_mock.call_count, 1)
-            urlpatterns[0].callback(**self.fake_post_kwargs)
-            self.assertEqual(csrf_decorator_mock.call_count, 2)
+                as csrf_mock:
```

Comment:
perhaps call this `csrf_exempt_mock`?: Without `exempt` the meaning might seem to be reversed.